### ISSUE TITLE
Да се генерализират функциите за създаване/обучаване на модел

### ISSUE THREAD

**Issue by IliyaKarkamov** [MariyaZhelyazkova/NN-UniversityProject#2]
в момента много неща в тях са хардкоднати, например размера на снимките и т.н. Хубаво е тези неща да излязат отвън.

**Comment by alefremov** [comment]
Да, това ще е добре да се изведе... Може и други неща да се подобрят, които са свързани с процеса на обучение. Не знам доколко имаме свобода да управляваме процеса, например можем ли да обучаваме само последния слой и т.н.
Ти избрал ли си тема за дипломна работа. Доколкото си спомням Мария вече е избрала, но не мога да се сетя дали с теб сме говорили. :)
